pkg/kubelet/server/server.go

```diff
@@ -138,7 +138,7 @@ func ListenAndServeKubeletServer(
 		// Passing empty strings as the cert and key files means no
 		// cert/keys are specified and GetCertificate in the TLSConfig
 		// should be called instead.
-		glog.Fatal(s.ListenAndServeTLS(tlsOptions.CertFile, tlsOptions.KeyFile))
+		glog.Fatal(s.ListenAndServeTLS("", ""))
```

Comment:
Why changing this?

Reply:
This is changed to force the usage of GetCertificates.  If certs are given then GetCertificates will not be called and we would need GetCertificates to be called.